mods/simple_service.js

```diff
@@ -325,7 +326,14 @@ SimpleService.prototype.processSpec = function(spec) {
                 var context = {
                     request: req
                 };
-                return generatePromiseChain(restbase, context, requestChain);
+                return generatePromiseChain(restbase, context, requestChain)
+                .then(function(res) {
+                    if (self.cache && res && /^2..$/.test('' + res.status)) {
```

Comment:
This will need some rework before we can enable it in prod. Could you add a todo comment to that effect, so that we don't forget to re-do this?